cdc/puller/sorter/sorter_test.go

```diff
@@ -76,7 +76,7 @@ func (s *sorterSuite) TestSorterBasic(c *check.C) {
 	sorter, err := NewUnifiedSorter("/tmp/sorter", "test-cf", "test", 0, "0.0.0.0:0")
 	c.Assert(err, check.IsNil)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
```

Comment:
Is there any CI link that shows this test is unstable?
It seems `FAIL: sorter_test.go:323: sorterSuite.TestSorterIOError` is an unstable case.

Reply:
the error I found:
```
[2021-06-01T02:58:29.766Z] FAIL: sorter_test.go:60: sorterSuite.TestSorterBasic
[2021-06-01T02:58:29.766Z] 
[2021-06-01T02:58:29.766Z] sorter_test.go:82:
[2021-06-01T02:58:29.766Z]     c.Assert(err, check.ErrorMatches, ".*context cancel.*")
[2021-06-01T02:58:29.766Z] ... error string = "context deadline exceeded"
[2021-06-01T02:58:29.766Z] ... regex string = ".*context cancel.*"
```

Reply:
Could we estimate how long this test could cost, is this an unknown bug or just slow environment causes slow case.
PTAL @liuzix

Reply:
`SorterBasic` timing out is likely due to slow environment. This case is not testing any error handling or edge cases.

Reply:
the same case fail at my side several times, is this change ready to merge